Makefile

```diff
@@ -314,20 +314,20 @@ bench-daily:
 
 build_tools: build_br build_lightning build_lightning-ctl
 
-br_web:
-	@cd br/web && npm install && npm run build
+lightning_web:
```

Comment:
also need to notice CD team about the makefile entry changes

Reply:
seems it's used by developers to generate some files, like `checkpoints.pb.go` and `res_vfsdata.go`

Reply:
@D3Hunter 
CD will call `build_tools` task, so keep the command and output same will ensure the CD work well.


Thank you @lance6716 !